.github/actions/coverage-report/action.yml

```diff
@@ -0,0 +1,104 @@
+name: Coverage Report
+
```

Comment:
this file seems to be only for Unix?
If so then worth rename?

Reply:
I think the OS it runs on is an implementation detail - it's generating test coverage. We do something similar in Hazelcast - i.e. test by different OS/JDK, but then we generate Sonar we just use Linux.

Reply:
ok. Its just that it calls `./scripts/test-unix.sh` so it is rerunning tests to get coverage on Unix and not for Windows so thought rename but no issue